crates/amaru-stores/Cargo.toml

```diff
@@ -25,6 +25,7 @@ amaru-ledger.workspace = true
 amaru-iter-borrow.workspace = true
 amaru-ouroboros-traits.workspace = true
 amaru-slot-arithmetic.workspace = true
+lazy_static = "1.5.0"
```

Comment:
`lazy_static` has been deprecated for `once_cell`, itself now replaced by std `LazyLock`.

Also, we don't define crate versions per amaru crate but globally.